src/cargo/util/config/mod.rs

```diff
@@ -176,6 +176,8 @@ pub struct Config {
     build_config: LazyCell<CargoBuildConfig>,
     target_cfgs: LazyCell<Vec<(String, TargetCfgConfig)>>,
     doc_extern_map: LazyCell<RustdocExternMap>,
+    /// Cache parsed manifests to avoid parsing twice.
+    manifest_cache: LazyCell<RefCell<cargo_toml::ManifestCache>>,
```

Comment:
This is ok to not store in a `LazyCell`, creating an empty cache is a cheap operation.